### PR TITLE
i18n(hi): add index page for hindi language

### PR DESCRIPTION
**Pull request by at-the-vr**
#### What kind of changes does this PR include?

- Adding first translated page for Hindi language

#### Description
- new index page added for Hindi language
- major config/yml/json files updated to include hindi language text

_Preview Deployment - [link](https://starlight-l5fotx2t5-astrodotbuild.vercel.app/hi/)

### CODE REVIEW

**Review by delucis — COMMENTED** [review]
Thank you for kicking this off @at-the-vr! I think I found the issue.

**Comment by at-the-vr** [comment]
Ofcourse all the relative paths were meant to be one level deep 👍

**Comment by at-the-vr** [comment]
omg its working, i18n is so cool

**Comment by at-the-vr** [comment]
> omg its working, i18n is so cool. missing some translations on the index page, I will follow up on this a bit later since its midnight here

follow up sent, deployment is active - [link](https://starlight-l5fotx2t5-astrodotbuild.vercel.app/hi/)

File: docs/src/content/docs/hi/index.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+hero:
+  title: अपने दस्तावेज़ों को Starlight से चमकाएँ
+  tagline: एक तारकीय दस्तावेज़ीकरण वेबसाइट बनाने के लिए आपको जो कुछ भी चाहिए वह सब कुछ इधर हैं। तेज़, अभिगम्य और उपयोग में आसान।
+  image:
+    file: ../../../assets/hero-star.webp
+  actions:
+    - text: शुरू हो जाओ
+      icon: right-arrow
+      variant: primary
+      link: /getting-started/
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/index.mdx:28]:
```suggestion
      link: /hi/getting-started/
```

This should point to the translated version even if it does not exist yet.

**Review by HiDeoo — DISMISSED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and everything looks pretty good. I only left a small comment regarding a link that should be updated.

**Review by delucis — APPROVED** [review]
Nice work @at-the-vr and thanks for catching that link @HiDeoo — I think this is good to go. Hindi docs launching now! 🚀

**Pull request merged**